addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -2368,6 +2372,15 @@ const Wysiwyg = Widget.extend({
                 this.linkPopover.hide();
             }
         }
+        const column = closestElement(this.currentTd, 'td');
+        const row = closestElement(column, 'tr');
+        const tableColumn = row && row.querySelector('td');
+        const isFirstColumn = column === tableColumn;
+        const table = column && closestElement(column, 'table');
+        const tableRow = table && table.querySelector('tr');
+        const isFirstRow = table && row === tableRow;
+
+        this.odooEditor._toggleTableUi(isFirstColumn && row, isFirstRow && column);
```

Comment:
Having to call private methods on the instance should be a red flag that this code is probably not in the right place.